app/Resources/views/subscription/tabs/tab-metadata.html.twig

```diff
@@ -1,24 +1,30 @@
 <fieldset>
     <div class="row">
         <div class="col-xs-11">
+            {{ form_row(form.importUrl) }}
             {{
                 form_row(
-                    form.metadataUrl,
+                    form.import,
                     {
-                        attr: {
-                            'class': 'url-validated',
-                            'data-parsley-remote': path(
-                                'validate',
-                                {id: subscription.id}
-                            ),
-                            'data-parsley-remote-options': '{ "type": "POST" }',
-                            'data-parsley-errors-messages-disabled': 'true'
+                        'attr': {
+                            'data-requestedState': 'import'
                         }
                     }
                 )
             }}
         </div>
-
+        <div class="col-xs-1">
+            {{
+                form_widget(
+                    form.import,
```

Comment:
Ja, maar in het geval van een submit button doet het ook niets meer dan dat, heeft geen label of errors.
Maar ik probeer om die stomme knop naast het import URL veld te krijgen maar dat wil niet lukken 😢